extensions/terminal-suggest/src/helpers/executable.ts

```diff
@@ -21,14 +21,16 @@ export async function isExecutable(filePath: string, configuredWindowsExecutable
 	}
 }
 
-function resolveWindowsExecutableExtensions(configuredWindowsExecutableExtensions: Object): string[] {
+function resolveWindowsExecutableExtensions(configuredWindowsExecutableExtensions?: Object): string[] {
 	const resolvedWindowsExecutableExtensions: string[] = windowsDefaultExecutableExtensions;
 	const excluded = new Set<string>();
-	for (const [key, value] of Object.entries(configuredWindowsExecutableExtensions)) {
-		if (value === true) {
-			resolvedWindowsExecutableExtensions.push(key);
-		} else {
-			excluded.add(key);
+	if (configuredWindowsExecutableExtensions) {
```

Comment:
If that's not set for some reason we still want to continue, just with everything I guess?

Reply:
Yes, that's what happens. https://github.com/microsoft/vscode/blob/d427cae0d59309c8bc367d8f93fb8c6e91b8e7de/extensions/terminal-suggest/src/helpers/executable.ts#L36

Reply:
see where that's set https://github.com/microsoft/vscode/blob/d427cae0d59309c8bc367d8f93fb8c6e91b8e7de/extensions/terminal-suggest/src/helpers/executable.ts#L25